default.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import ./pkgs.nix {}, shell ? false }:
+{ pkgs ? import ./nixpkgs.nix {} }:
```

Comment:
@zo-el the original PR did't remove this `shell ? false` expression, that was something I did when resolving merge conflicts. Wasn't sure the right thing to do, maybe we want to keep that expression?

Reply:
We do need that. It will give you access to `sim2h_server` cli